src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java

```diff
@@ -487,4 +491,19 @@ public void afterNavigateForward(WebDriver driver) {
   public void onFinish(ITestContext context) {
     CommonUtils.appendTextToFile(logPath, "</body></html>");
   }
+
+  private void checkCountryCode(String expectedCountryCode) {
+    if (StringUtils.isBlank(expectedCountryCode)) {
+      return;
+    }
+
+    String actualCountryCode = new AdsBaseObject(driver).getCountry();
```

Comment:
It's a little bit strange that `checkCountryCode()` depends on `AdsBaseObject`. Do you think we can do cleaner? Ping me if you need ideas, please :smile_cat: